spec/services/taxonomy/taxonomy_query_spec.rb

```diff
@@ -3,11 +3,25 @@
 
 include Taxonomy
 include ::GdsApi::TestHelpers::ContentStore
+include ::GdsApi::TestHelpers::PublishingApiV2
 
 RSpec.describe Taxonomy::TaxonomyQuery do
   def query
     TaxonomyQuery.new(%w[content_id base_path])
   end
+
+  describe '#parent' do
+    it 'returns nil because the taxon is a level one taxon' do
+      publishing_api_has_expanded_links('content_id' => 'f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a', 'expanded_links' => {})
+      expect(query.parent('f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a')).to be_nil
+    end
+    it 'returns a parent' do
+      publishing_api_has_expanded_links(expanded_links)
+      expect(query.parent('f3bbdec2-0e62-4520-a7fd-6ffd5d36e03a')).to eq('base_path' => "/path/base",
```

Comment:
You're using the content_id of the root taxon again in this test.  I'd use another content_id to illustrate the point of the test.

Reply:
fixed